content/docs/reference/distribution-api.md

```diff
@@ -0,0 +1,25 @@
++++
+title="Distribution API"
+weight=2
+creatordisplayname = "Joe Kutner"
+creatoremail = "jpkutner@gmail.com"
+lastmodifierdisplayname = "Joe Kutner"
+lastmodifieremail = "jpkutner@gmail.com"
++++
+
+This specification defines the artifact format, delivery mechanism, and order resolution process for buildpacks.
+
+## Buildpack Descriptor
+
+A buildpack must contain a `buildpack.toml` file in its root directory.
+
+## Buildpackage
+
+A buildpackage is a distributable artifact that contains a buildpack. Its format may be either:
```

Comment:
Can it contain multiple buildpacks? (Or perhaps if the answer is: yes, via an order buildpack, then maybe we should explain how to do that in the buildpack doc).

Reply:
I was hoping to avoid any complex discussion on order buildpacks here by referring to the singular "buildpack" (which is still accurate). But maybe it's just more confusing without a mention?

Reply:
That makes sense